### PR TITLE
Kiwix lib app

### PR DESCRIPTION
**Pull request by mgautierfr**

### CODE REVIEW

File: scripts/publish_to_bintray.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import requests
+from requests.auth import HTTPBasicAuth
+import os, sys
+import json
+
+bintray_auth = (os.environ['BINTRAY_USER'], os.environ['BINTRAY_PASS'])
+
+
+def create_version(version):
```
Comment (macgills) [inline scripts/publish_to_bintray.py]: https://github.com/kiwix/kiwix-lib/blob/aa7f61c354ae6ab28829696e84e1cd7f29cf520a/android-kiwix-lib-publisher/kiwixLibAndroid/publish.gradle I am not seeing the depth of information that we used to be supplying here.
We - I think - have to be supplying the groupId at least and the artefact name.
I think the generation of a pom file is necessary? To be honest this is the first artefact I've uploaded so I don't have huge knowledge here

Comment (mgautierfr) [reply scripts/publish_to_bintray.py]: The new version of the script also publish the pom file.

File: scripts/publish_to_bintray.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      <name>kiwix</name>
+      <email>contact@kiwix.org</email>
+    </developer>
+  </developers>
+  <scm>
+    <connection>https://github.com/kiwix/kiwix-lib.git</connection>
+    <developerConnection>https://github.com/kiwix/kiwix-lib.git</developerConnection>
+    <url>https://github.com/kiwix/kiwix-lib</url>
+  </scm>
+  <dependencies>
```
Comment (macgills) [inline scripts/publish_to_bintray.py]: I don't like this dependency being here, I thought I had removed that.
Logistically the project has no dependencies I thought

**Comment by mgautierfr** [comment]
Note that with this PR, the CI doesn't build kiwix-android anymore. It only build kiwix-lib for android.
This should fix https://github.com/kiwix/kiwix-android/pull/1234#issuecomment-506148260

File: scripts/publish_to_bintray.py (hunk 2 above)
Comment (mgautierfr) [reply scripts/publish_to_bintray.py]: This dependency is present in kiwix-lib 1.0.12.
But I can remove it if you think it is not necessary.

Comment (macgills) [reply scripts/publish_to_bintray.py]: It was probably a mistake, should be rectified in this https://github.com/kiwix/kiwix-lib/pull/231 project which is a requirement to be merged before this is valid, should we switch context back to getting that merged?

Comment (mgautierfr) [reply scripts/publish_to_bintray.py]: I've just removed the dependency.
(And also fix the license to GPLv3)

File: scripts/publish_to_bintray.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
```
Comment (macgills) [inline scripts/publish_to_bintray.py]: maybe rename this file, it is not very generalised so "upload_kiwix_lib_android_to_bintray" might help understanding

Comment (mgautierfr) [reply scripts/publish_to_bintray.py]: Done

**Comment by kelson42** [comment]
@macgills @mgautierfr Which ticket fixes this PR? What is the status?

**Comment by mgautierfr** [comment]
This is the kiwix-build side of kiwix/kiwix-lib#231

**Comment by mgautierfr** [comment]
It seems there is an issue with appveyor reporting the build status.
It is ok (https://ci.appveyor.com/project/Kiwix/kiwix-build/builds/26203698), I'm merging this PR.

**Pull request merged**